sim/warrior/arms/talents.go

```diff
@@ -122,9 +122,6 @@ func (war *ArmsWarrior) TriggerSlaughter(sim *core.Simulation, target *core.Unit
 
 	rend := war.Rend.Dot(target)
 	if rend != nil && rend.IsActive() {
-		if rend.NextTickAt() == sim.CurrentTime {
```

Comment:
Yeah, the old dot code had a check in there that would prevent it from ever double ticking, as a hacky fix to work around a different problem.

I've fixed the underlying issue so that should not be a problem anymore, and thus also removed this.